scripts/asr_language_modelling/ngram_lm/train_kenlm.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Rename to `ngram_kenlm_subword/train_subword_kenlm.py` to make it clear this is for subword CTC models only.

Reply:
added the support for char-level encoding.